models/sources/src_accounts.yml

```diff
@@ -14,7 +14,6 @@ sources:
           - incremental_unique_combination_of_columns:
               combination_of_columns:
                 - account_id
-                - sequence_number
```

Comment:
I think `sequence_number` should be included. The src/staging `accounts` table should record every instance of an account on every `sequence_number` iirc

Reply:
Got it. I will keep it then, It was not present in the scheduled query, so I treated that as source of truth

Reply:
Addressed in https://github.com/stellar/stellar-dbt-public/pull/83/commits/6bd6abf411f043b3e30af1bfa1780741b9d9cac1